profiles/shivamp1998.json

```
@@ -0,0 +1,6 @@
+{
+    "image" : "https://user-images.githubusercontent.com/48366063/193478291-efc11969-6ba4-41bb-bfdf-89d1644afdba.png",
+    "issueId" : 1454,
+    "name" : "sarvesh pandey",
+    "username" : "shivamp1998"
```

It's best to remove the spaces before `:`.
```suggestion
    "image": "https://user-images.githubusercontent.com/48366063/193478291-efc11969-6ba4-41bb-bfdf-89d1644afdba.png",
    "issueId": 1454,
    "name": "sarvesh pandey",
    "username": "shivamp1998"
```

---

Done, please review!